lib/adagrams.rb

```diff
@@ -0,0 +1,90 @@
+require "awesome_print"
+
+def draw_letters
+  avail_letters = %w[A A A A A A A A A B B C C D D D D E E E E E E E E E E E E F F G G G H H I I I I I I I I I J K L L L L M M N N N N N N O O O O O O O O P P Q R R R R R R S S S S T T T T T T U U U U V V W W X Y Y Z]
+  avail_letters.sample(10)
```

Comment:
In general, your code takes advantage of Ruby's implicit `return` statements, but if you ever felt like going back to explicit `return` statements, I wouldn't be mad...